File: swagger/models/get_locations_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetLocationsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		locations
		Contains information about the locations.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse)
		'Locations': 'locations',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'locations': 'Locations',
		}

File: swagger/Models/GetLocationsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetLocationsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the locations.
	 * 
	 * @var Location[] 
	 */
	public $locations;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Locations' => ['locations', 'array', Location::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'locations' => ['Locations', 'array', Location::class],
			];
	}
}
